index.html

```diff
@@ -0,0 +1,334 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <script src="https://use.fontawesome.com/23bea62652.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">
+    <title>Evolo</title>
+</head>
+<body>
+
+
+    <!--Section1: Design And Plan Your Business Growth Steps -->
+    <section class="services1">
+        <div class="container">
+            <div class="row">
+                <div class="col1">
+                    <div class="text-container">
+                        <h2>Design And Plan Your Business Growth Steps</h2>
+                        <p>Use our staff and our expertise to design and plan
+                           your business growth strategy. Evolo team is eager
+                           to advise you on the best opportunities that you should
+                           look into</p>
+                        <a href="#" class="services-btn">LIGHTBOX</a>
+                    </div>
+                </div>
+                <div class="col2">
+                    <div class="img-container">
+                        <img class="img-fluid" src="./images/details-1-office-worker.svg" alt="website template image"
+                        width="540px" height="453px">
```

Comment:
instead of the inline style here(width, height), it's better to put all styles on `style.css` file, to seperate HTML and CSS.